markdown-export/render-rules.ts

```diff
@@ -16,6 +16,44 @@ interface TableCell {
     align: string | undefined;
 }
 
+function parseAndFormatDate(str: string): string {
+    // TW date format (spaces added for clarity): [UTC] YYYY 0MM 0DD 0hh 0mm 0ss 0XXX
+    const regex = /^(\d{4})(\d{2})(\d{2})(\d{2})(\d{2})(\d{2})(\d{3})$/;
+    const match = str.match(regex);
+    if (match) {
+        const isoDateString = `${match[1]}-${match[2]}-${match[3]}T${match[4]}:${match[5]}:${match[6]}Z`;
+        const date = new Date(isoDateString);
```

Comment:
I believe this can be achieved with `new Date($tw.utils.parseDate(str))` -- but I'm not sure. TiddlyWiki has functions to parse and convert dates.

Reply:
I'll take a look into this, thanks for pointing it out. Probably better to use TW functions if we can. Will experiment with it...

Reply:
Based on testing seems TW `parseDate()` is the way to go so was able to get rid of what was redundant code. :-)